monster-cards/src/styles/css/font-awesome.css

```diff
@@ -0,0 +1,2199 @@
+/*!
```

Comment:
Do we usually add 3rd party code into sub-dirs?
I would have thought that adding FA as a `npm` dep and then moving into sub-dirs when packaging would be better for maintainability, e.g. you can update the FA version by changing `package.json` rather than wholesale dir replacement (FA changes pretty often).